problemsets-mat223/modules/module14.tex

```diff
@@ -273,15 +273,22 @@
 \begin{example}
 	Use elementary matrices to find the determinant of $A=\mat{1&2\\3&4}$.
 
-	By applying row reduction to $A$, we could obtain
-	\[
-	    \mat{1&-2\\0&1}\mat{1&0\\0&-\frac{1}{2}}\mat{1&0\\-3&1}\mat{1&2\\3&4} = \mat{1&0\\0&1}.
-	\]
-	Therefore, by the theorem above, we would get
+	We can row-reduce $A$ with the following steps:
+	\begin{align*}
+	    \mat{1&2\\3&4}&\sim \mat{1&2\\0&-2}\\
+	                  &\sim \mat{1&2\\0&1}\\
+	                  &\sim \mat{1&0\\0&1} 
+	\end{align*}
+	Thus we could obtain
```

Comment:
Since this is the first time actually seeing this, let's spell it out very explicitly. "The elementary matrices corresponding to these steps are E_1=..., E_2=..., and \[ E_3E_2E_1A=I. \] Therefore \[A=E_1^-1E_2^-1E_3^-1.\]

Reply:
Remove `could`.